python/mxnet/ndarray/random.py

```diff
@@ -518,3 +518,47 @@ def shuffle(data, **kwargs):
     <NDArray 2x3 @cpu(0)>
     """
     return _internal._shuffle(data, **kwargs)
+
+
+def randint(low=0, high=1, shape=_Null, dtype=_Null, ctx=None, out=None, **kwargs):
+    """Draw random samples from a discrete uniform distribution.
+
+    Samples are uniformly distributed over the half-open interval *[low, high)*
+    (includes *low*, but excludes *high*).
+
+    Parameters
```

Comment:
the description needs to tell which of the parameters are required and which are optional.

Reply:
I followed the template of all the other random functions. None of them do it at the moment. Do you want me to add that to all of them?

Reply:
yes, that would be good.